docs/quality-assurance/Web-Accessibility.md

```diff
@@ -0,0 +1,26 @@
+# Web Accessibility Testing
+
+Accessibility is not a 'nice to have', it's a legal requirement for companies and organisations to be inclusive for everyone.
+
+Accessibility Testing is not confirmation-based. Instead of testing the functionality works, testers need to look for how it **can't** be used because of an accessibility barrier.
+
+Automated tools include:
+
+- core installed in delivery pipeline - https://www.deque.com/axe/
+- Lighthouse - https://developers.google.com/web/tools/lighthouse
+
+Accessibility Testing is 'hands on'. While automated tests will catch things like missing alt text, contrast problems, and missing form labels; they are no substitute for manual hands on testing. Pre-written manual testing scripts cannot be relied upon because they provide no more insight than automated testing. The tester must explore different ways of using the application in order to tease out problems with accessibility.
+
+Manual Test Tools include:
```

Comment:
I personally feel there could be more guidance around the tooling. e.g. why would you use one other the over? Should we be using more than one? advantages / disadvantages? a comparison table with each tool and what it verifies?